docs/getting-started-guides/vsphere.md

```diff
@@ -5,13 +5,47 @@ assignees:
 
 ---
 
+* TOC
+{:toc}
+
+## vSphere Cloud Provider
+
+On a vSphere cluster it is recommended to use the vSphere cloud provider. vSphere cloud provider currently supports 
+
+1. [Volumes](http://kubernetes.io/docs/user-guide/volumes)
+2. [Persistent Volumes](http://kubernetes.io/docs/user-guide/persistent-volumes/)
+   1. [Static](http://kubernetes.io/docs/user-guide/persistent-volumes/#static)
+   2. [Dynamic](http://kubernetes.io/docs/user-guide/persistent-volumes/#dynamic)
+3. [Storage Class](http://kubernetes.io/docs/user-guide/persistent-volumes/#storageclasses)
+
+## Configuring vSphere Cloud Provider
+
```

Comment:
Please add steps:
1. Create Cloud Configuration file on Master node. e.g. .etc/vsphere-cloud.conf
2. Document parameters in cloud config
3. Add command line demonstrating how to pass cloud-config to kube executables

I assume all of these steps are not needed if cluster is provisioned using kube-up